turbinia/evidence.py

```diff
@@ -259,6 +260,18 @@ class Directory(Evidence):
   pass
 
 
+class CompressedDirectory(Evidence):
+  """A compressed directory containing data."""
+
+  def __init__(self, *args, **kwargs):
+    super(CompressedDirectory, self).__init__(*args, **kwargs)
+    self.copyable = True
+
+  def _postprocess(self):
```

Comment:
Even if it's not going to be used yet, can we add the preprocess method to decompress the evidence as well?

Reply:
Added a _preprocess method to the CompressDirectory evidence object. Currently I have my error handling set to raise an exception when you pass a non tar file (aka a directory) to the Uncompress method, I could change this to just pass if a directory is provided.